js/layout-javascript/agenda-code.js

```diff
@@ -1653,6 +1656,10 @@ DynamicList.prototype.centerDate = function() {
   var activeWidth = _this.$container.find('.agenda-date-selector li.active').outerWidth();
   var halfWidth = activeWidth / 2;
 
+  if (!activePosition || isNaN(activePosition.left) || isNaN(halfWindowWidth) || isNaN(halfWidth)) {
```

Comment:
What else can `activePosition.left`, `halfWindowWidth` and `halfWidth` be instead of a number?

Reply:
If it's not a number we can't use it, hence why that if statement.

I just care that it is a number.

Reply:
Any concerns?
I know that if it is `null` it will give a false positive.

But from my tests it will never be `null` it can either be a `Number` or `undefined`

Reply:
Those were my concerns and that's why I asked but you summed it up. Thanks!